services/harmony/app/models/label.ts

```diff
@@ -27,22 +31,97 @@ export function normalizeLabel(label: string): string {
   return label.trim().toLowerCase();
 }
 
+/**
+ * Verify that the user can change the labels on a give job. Currently only job owners can
```

Comment:
job owners and admin can change labels for a job.

Reply:
fixed